dev/io.openliberty.jakarta.pages.tld.3.1/bnd.bnd

```diff
@@ -0,0 +1,40 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+-nouses=true
+
+Bundle-SymbolicName: io.openliberty.jakarta.pages.tld.3.1
+Bundle-Description: Jakarta Standard Tag Library, version 3.0
```

Comment:
We should update this to version 3.1, or open an issue so we don't forget. I'm fine either way just let me know.